distributed/tests/test_cancelled_state.py

```diff
@@ -169,10 +169,10 @@ async def wait_and_raise(*args, **kwargs):
     # refactoring. Below verifies some implementation specific test assumptions
 
     story = w.story(fut.key)
-    start_finish = [(msg[1], msg[2]) for msg in story if len(msg) == 6]
-    assert ("executing", "cancelled") in start_finish
-    assert ("cancelled", "error") in start_finish
-    assert ("error", "released") in start_finish
+    start_finish = [(msg[1], msg[2], msg[3]) for msg in story if len(msg) == 7]
+    assert ("executing", "released", "cancelled") in start_finish
+    assert ("cancelled", "error", "error") in start_finish
+    assert ("error", "released", "released") in start_finish
```

Comment:
This should've failed hard in https://github.com/dask/distributed/pull/5525 already but we likely just missed it due to all the failure noise lately